mail-toaster.sh

```diff
@@ -535,6 +539,7 @@ has_data_fs()
 		postgres ) return 0;;
 		haproxy )  return 0;;
 		minecraft ) return 0;;
+		php7 )     return 0;;
```

Comment:
Once upon a time, I started down the "install PHP in one jail, squirrelmail in another, roundcube in another, nginx in another" road. I very much liked that idea, but somewhere down that road I ran into an obstacle so I retreated back to the current approach.

I think perhaps that was before I worked out the current data volumes. Now that that those are in place, my inclination is that instead of having a `php7` jail, have a `php` jail that builds either PHP5 or PHP7. The question is then becomes, might one need a `php5` jail and a `php7` jail?  In cases like that, the software that has the dependency should just have its own version of PHP installed along with it (squirrelmail, roundcube, mediawiki, etc...), and all the others can used the standalone instance. Thoughts?